spec/lib/ansible/runner/credential/machine_credential_spec.rb

```diff
@@ -101,6 +101,40 @@ def password_hash
 
         expect(password_hash["^SSH [pP]assword:"]).to eq(password)
       end
+
+      context "with an existing password_file" do
+        let(:ssh_unlock_key) { "^Enter passphrase for [a-zA-Z0-9\-\/]+\/ssh_key_data:" }
+        def existing_env_password_file(data)
+          cred # initialize the dir
+          File.write password_file, data.to_yaml
```

Comment:
Can you add parenthesis here?

Reply:
I thought the bot would have complained about this ... :confused:

Reply:
It doesn't complain in `spec/` because we allow for things like this:

```ruby
expect(true).to be_truthy
```

Otherwise the bot would require it to be like this:

```ruby
expect(true).to(be_truthy)
```